tests/garage/tf/algos/test_ddpg.py

```diff
@@ -52,3 +52,5 @@ def test_ddpg_pendulum(self):
         )
         last_avg_ret = algo.train(sess=self.sess)
         assert last_avg_ret > 40
+
+    test_ddpg_pendulum.flaky = True
```

Comment:
it seems that the easier way to fix this test is to pick a better value for `last_avg_ret` rather than turning if off entirely.

Reply:
I think 30 will be a safe choice. I'll assert the algorithm is not working if lower than 30.